domains/idkmaxi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Molret",
+        "email": "maximiliano.moldenhauer.r@marygraham.cl"
+    },
+    "record": {
+        "CNAME": "ns1.infinityfree.com"
```

Comment:
You can't use namerserver records (NS) and it wouldn't work if u set it as CNAME